mllib/src/test/scala/org/apache/spark/ml/feature/CountVectorizerSuite.scala

```diff
@@ -183,6 +183,26 @@ class CountVectorizerSuite extends SparkFunSuite with MLlibTestSparkContext
       case Row(features: Vector, expected: Vector) =>
         assert(features ~== expected absTol 1e-14)
     }
+
+    // CountVectorizer test
+    val df2 = sqlContext.createDataFrame(Seq(
```

Comment:
if you create a model from a given vocab, I believe the expected transform would be the same as a model that is fit, as long as same tokens in the 2 vocabs have the same indicies - so using a vocab with an added "d" token should be fine.